index.js

```diff
@@ -76,7 +76,6 @@ module.exports = {
     "selector-attribute-operator-space-before": "never",
     "selector-combinator-space-after": "always",
     "selector-combinator-space-before": "always",
-    "selector-class-pattern": "(^d2l-)|(^focus-visible$)",
```

Comment:
What does this have to do with shadow DOM? I thought it was defining the CSS class selector pattern that we want folks to use -- i.e. `.d2l-foo` instead of `.foo`.

Reply:
I think we wanted them to do that because of the potential collision that could occur when shadyDOM was in play.  By prefixing their classes with the component name, it was preventing CSS defined in the document from targeting HTML elements in the shadyDOM.  Now that all the components' elements are properly scoped, we should be able to use more succinct, less redundant class names.  If we want to keep the rule for conventional reasons, I don't feel strongly about it, but this linting rule was originally added to avoid collision.

Reply:
By "preventing", what I mean is, it was reducing the likelihood of a collision.

Reply:
Ah I understand. Yeah I guess that's definitely no longer a requirement and I also don't feel super strongly about keeping it around as a convention. 👍